src/routes/userRouter.js

```diff
@@ -10,10 +10,276 @@ const {
   login,
 } = require("../controllers/userController");
 
+/**
```

Comment:
Great job documenting the api's, this will def help the frontend folks!

When you're working with Swagger for API documentation, it's a great idea to keep things tidy by placing the documentation right alongside the code it describes.

For example, if you're documenting a login endpoint, put the Swagger info directly into the `Login.js` file where you've got your login API set up. That way, Swagger can pick it up automatically and show it in the right spot in your API documentation.

However, it seems like we have both login and sign up api definitions within the User controller. Separating functionality helps for situations like these and I think adding swagger files there may make thing harder to understand.

I say keep the swagger docs as you have them! Ship it! 🚢

Reply:
Thank you! I will keep this in mind and implement it in future projects.